recipes/qbs/all/conandata.yml

```diff
@@ -0,0 +1,46 @@
+sources:
+  "1.16.0":
+    Linux:
```

Comment:
The `conandata.yml` file does not allows custom fields, so none of this will work

Reply:
[But it works](https://travis-ci.com/github/rweickelt/conan-center-index).

[And here is why](https://docs.conan.io/en/latest/reference/config_files/conandata.yml.html):
> This file can be used, for example, to declare a list of sources links and checksums for the recipe or a list patches to apply to them, but you can use it to store any data you want to extract from the recipe. For example: 

Or am I missing something?

Reply:
OK, it's a restriction by conan-center. Should be fine now.